api/HTMLAreaElement.json

```diff
@@ -253,10 +253,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": false
```

Comment:
Removing this `media` entry in https://github.com/mdn/browser-compat-data/pull/7538.